test/zuliprc

```diff
@@ -1,4 +1,4 @@
 [api]
-email=cordelia@zulip.com
-key=wlueAg7cQXqKpUgIaPP3dmF4vibZXal7
-site=http://localhost:9991
+email=api-test-bot@chat.zulip.org
+key=K1w7xuUlCorRXjDCz2EkCC4IXMbdYOpK
+site=https://chat.zulip.org
```

Comment:
To do that would be significant setup work -- we'd need to basically run the main Zulip server project's development environment provisioning.

I guess the way to make that easy would be to merge this project into zulip.git.

Do the tests fail if you just include the changes actually related to the feature in this PR?